### DISCUSSION TITLE
Next localization cookie

### DISCUSSION THREAD

**Discussion by ivandotv** [vercel/next.js#20038] [Help] [answered]
I see that in i18n Routing RFC there is a mention of `routing cookie`
> A Locale can be forced by setting the NEXT_LOCALE cookie to a locale that is supported by the application. E.g. NEXT_LOCALE=nl-NL

I have tested this functionality with the latest published version of the framework however, it is not in the documentation, can we rely on this functionality?

**Comment by ijjk** [comment, marked as answer]
Hi, this can be relied on and is fully supported, I opened a PR here https://github.com/vercel/next.js/pull/20045 adding this to our documentation, thanks for bringing this to our attention!
